classes/Http/Controller/Admin/SpeakersController.php

```diff
@@ -165,20 +178,11 @@ public function deleteAction(Request $request)
 
     public function demoteAction(Request $request)
     {
-        /** @var AccountManagement $accounts */
-        $accounts = $this->service(AccountManagement::class);
-
         $role = $request->get('role');
         $id   = (int) $request->get('id');
 
-        /** @var Authentication $authentication */
-        $authentication = $this->service(Authentication::class);
-
-        /** @var Session\Session $session */
-        $session = $this->service('session');
```

Comment:
Since we would otherwise be invoking `$request->getSession()` multiple times, can we maybe just use `$session` here and assign it `$request->getSession()`?

Reply:
`$request->getSession()` isn't an expensive operation, but yes, I can change that.

Reply:
On the other hand: `getSession()` is usually followed by a `return` statement, so we're not really invoking `getSession()` multiple times. I wouldn't change it, tbh.